src/mqtt.base-client.ts

```diff
@@ -117,29 +139,14 @@ export class MqttBaseClient<
         this.next(StateId.Fatal);
     }
 
-    private _connectResolver?: Resolvable<RegisterClientOptions>;
```

Comment:
Have no idea why these changes were added in this commit. I have not made them at all.